fluent-package/msi/assets/fluent-package-post-migration.bat

```diff
@@ -0,0 +1,66 @@
+@echo on
+title Fluent-package post migration script
+
+if exist "%~dp0..\..\td-agent" (
+  setlocal enabledelayedexpansion
+  sc query state=inactive | findstr fluentdwinsvc
+  if !ERRORLEVEL! == 1 (
+    echo "Active fluentdwinsvc service is detected"
+    net stop fluentdwinsvc
+  )
+  endlocal
+
+  echo "Migrate c:\opt\td-agent\etc\td-agent.conf"
+  if exist "%~dp0..\..\td-agent\etc\td-agent\td-agent.conf" (
+     move /Y "%~dp0..\..\td-agent\etc\td-agent\td-agent.conf" "%~dp0..\etc\fluent\td-agent.conf"
+     echo "Migrate td-agent.conf to c:\opt\fluent\etc\fluent\fluentd.conf"
+     copy /Y "%~dp0..\etc\fluent\td-agent.conf" "%~dp0..\etc\fluent\fluentd.conf"
+  )
+  echo "Create c:\opt\fluent\var\log\fluent directory"
+  if not exist "%~dp0..\var\log\fluent" (
+     mkdir "%~dp0..\var\log\fluent"
+  )
+  echo "Migrate c:\opt\td-agent\var\log\td-agent\* files"
+  for %%f in (%~dp0..\..\td-agent\var\log\td-agent\*) do (
+     move /Y %%f "%~dp0..\var\log\fluent\"
+  )
+  echo "Migrate c:\opt\td-agent\var\log\td-agent\* directory (e.g buffer)"
+  for /d %%d in (%~dp0..\..\td-agent\var\log\td-agent\*) do (
+     move /Y %%d "%~dp0..\var\log\fluent\"
+  )
+  echo "Migrate c:\opt\td-agent\etc\td-agent\* files"
+  for %%f in (%~dp0..\..\td-agent\etc\td-agent\*) do (
+     move /Y %%f "%~dp0..\etc\fluent\"
+  )
+  echo "migrate c:\opt\td-agent\etc\plugin directory"
+  for /d %%d in (%~dp0..\..\td-agent\etc\plugin\*) do (
+     move /Y %%d "%~dp0..\etc\fluent\plugin\"
+  )
+  echo "Migrate c:\opt\td-agent\*.log files"
+  for %%f in (%~dp0..\..\td-agent\*.log) do (
+     move /Y %%f "%~dp0..\fluent\"
+  )

```

Comment:
Somehow, this doesn't seem to work.
I confirmed the package behavior, but I couldn't find the old log files.

I'm reconsidering log file migration on Windows.
On Windows, even if the log file is moved, Fluentd will output the log to new files.
So, we may not need to move the log file.
(It remains in the backup-ed directory `td-agent.old`.)